ethcore/src/miner/miner.rs

```diff
@@ -277,13 +276,19 @@ impl MinerService for Miner {
 				// give the sender max balance
 				state.sub_balance(&sender, &balance);
```

Comment:
Seems it needs a similar workaround as the code in `client`.
Isn't it possible to reuse the code here and in client?